node-services/cards-external-diffusion/config/default.yml

```
@@ -54,4 +54,7 @@ operatorfabric:
       activateCardsDiffusionRateLimiter: true
       sendRateLimit: 100
       sendRateLimitPeriodInSec : 3600
-      defaultTimeZone: 'Europe/Paris'
\ No newline at end of file
+      defaultTimeZone: 'Europe/Paris'
+      customConfig: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)